### DISCUSSION TITLE
dockge and uptime-kuma intergration

### DISCUSSION THREAD

**Discussion by huuscript** [louislam/dockge#601] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API, UI Feature, Other

### 🔖 Feature description

It would be very convenient if a feature can be added to have dockge pass on information to monitor the containers into uptime-kuma.

### ✔️ Solution

in uptime-kuma when add new monitor there is a pull down list “monitor type” add another type for dockge. when all credentials enter uptime-kuma can auto populate all docker containers and add tags “dockge”. This also have an added benefit to be able to notify dockage agent is down.

**Comment by richarvey** [comment]
I'd like to see at the very least a button in the top right hand navigation that links you through to your uptime-kuma page. You could make this optional and only appear if you have a UPTIME_KUMA_URL=https://uptime.example.org environment variable maybe?
